Makefile

```diff
@@ -25,7 +25,7 @@ OPENCV_VERSION=4.0.1
 OPENCV_ARCHIVE=$(OPENCV_BASENAME).tar.gz
 OPENCV_BASENAME=opencv-$(OPENCV_VERSION)
 OPENCV_BUILD_LIB_DIR=$(OPENCV_BASENAME)/build/lib
-OPENCV_BUILD=$(OPENCV_BUILD_LIB_DIR)/python$(PYTHON_MAJOR_VERSION)/cv2*.so
+OPENCV_BUILD=$(OPENCV_BUILD_LIB_DIR)/cv2*.so
```

Comment:
Good catch, but I think it's actually the line above that should be kept

Reply:
Do you mean that w should not use `OPENCV_BUILD` anymore ?

Reply:
No I mean you deleted the variable that we want to keep, the other should be removed and we should keep:
```
OPENCV_BUILD=$(OPENCV_BUILD_LIB_DIR)/python$(PYTHON_MAJOR_VERSION)/cv2*.so
```
Somehow at it on my compile `cv2` library is dumped in: `opencv-4.0.1/build/lib/python3/cv2.cpython-36m-x86_64-linux-gnu.so` same thing on Travis.
For instance see this recent build log:
https://api.travis-ci.org/v3/job/501134779/log.txt
```
Scanning dependencies of target opencv_python3
make[3]: Leaving directory '/home/user/opencv-4.0.1/build'
make[3]: Entering directory '/home/user/opencv-4.0.1/build'
[100%] Building CXX object modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o
[100%] Linking CXX executable ../../bin/opencv_interactive-calibration
make[3]: Leaving directory '/home/user/opencv-4.0.1/build'
[100%] Built target opencv_interactive-calibration
[100%] Linking CXX shared module ../../lib/python3/cv2.cpython-36m-x86_64-linux-gnu.so
make[3]: Leaving directory '/home/user/opencv-4.0.1/build'
[100%] Built target opencv_python3
make[2]: Leaving directory '/home/user/opencv-4.0.1/build'
make[1]: Leaving directory '/home/user/opencv-4.0.1/build'
cp opencv-4.0.1/build/lib/python3/cv2*.so venv/lib/python3.6/site-packages
```

Reply:
Ok, for some reason it doesn't output to the same path for me.

Reply:
Ouch OK that's annoying. For me as as well when it was on opencv2 it was in `build/lib/cv2*.so`.
Maybe we should try a hybrid approach with a comment explaining both path are somewhat possible. But I would keep that for later.

Reply:
Ok, found out that actuallly that there is `build/lib/cv2.so` and I thought this one was the lib we wanted to copy.

Reply:
Oh yes I remember I had the same issue when migrating to opencv4, so yes I also have that lib, but it's not the one being used at the end somehow. I didn't investigate why.
But good to know that you also have something in `opencv-4.0.1/build/lib/python3/` that way we can keep the Makefile "simple"